### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mmattioli**
Commit titles/messages are self-explanatory and straightforward; just a little tidying up :smile: I can push a commit that updates the CHANGELOG.md but there aren't any drastic changes so I'm not sure if it still warrants an update there, let me know and I'll be happy to do so.

- [ ] CHANGELOG.md updated
- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](http://influxdb.com/community/cla.html) (if not already signed)

### CODE REVIEW

File: cmd/influx/cli/cli.go [hunk 1]
```diff
@@ -210,8 +210,6 @@ func (c *CommandLine) ParseCommand(cmd string) error {
 		case "exit", "quit":
 			// signal the program to exit
 			close(c.Quit)
-		case "gopher":
```
Comment (gunnaraasen) [inline cmd/influx/cli/cli.go]: The gopher is an easter egg and should not be removed. :)

Comment (mmattioli) [inline cmd/influx/cli/cli.go]: Just pulled that commit out :+1:

**Comment by mmattioli** [comment]
@dgnorton @gunnaraasen not sure why the CircleCI tests failed? Could it be because I removed a commit after it started running the tests?

**Comment by corylanou** [comment]
We have some intermittent failures in our test suite we are still chasing down.  The only happen in circle so it's been problematic pin pointing the issue so far.  I kicked off the tests again to see if they pass.  Thanks!

**Comment by mmattioli** [comment]
@dgnorton @jwilder @corylanou is there anything else outstanding I need to complete for this PR? Thanks.

**Comment by corylanou** [comment]
+1

**Pull request merged**